.github/workflows/test_benchmarks.yml

```diff
@@ -61,7 +66,7 @@ jobs:
         miniforge-variant: Mambaforge
         use-mamba: true
         channels: conda-forge
-        python-version: 3.9
+        python-version: ${{ inputs.benchopt_version }}
```

Comment:
Oups too fast!

this should be `python_version`. Just fixed in 8a61f57793a8e01d15efb43dd49492172d06d4a6

This shows that we should modify the CI to run against the modify workflow when updating it... :(